lib/classes/PluginManager.js

```diff
@@ -35,12 +47,21 @@ class PluginManager {
 
     forEach(options, (value, key) => {
       if (value.required && (this.options[key] === true || !(this.options[key]))) {
-        throw new this.serverless.classes.Error(`Please provide the "${key}" option`);
+        const errorMessage =
+        `
```

Comment:
This newline can be removed once the automatic newline addition is done in the Error class.